.github/workflows/test-and-publish.yaml

```
@@ -0,0 +1,97 @@
+      with:
+        files: ./coverage.xml
+        fail_ci_if_error: true
+
+  publish:
+    name: Publish to PyPI
+    needs: [code-quality, test]
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v')
+
+    steps:
+    - uses: actions/checkout@v3
```

yamllint

[warning] wrong indentation: expected 6 but found 4

(indentation)